plugins/aws/rds/rdsDefaultPort.js

```diff
@@ -0,0 +1,73 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'RDS Default Port',
+    category: 'RDS',
+    domain: 'Databases',
+    description: 'Ensure RDS database instances are not using the default ports.',
+    more_info: 'Using default ports for running RDS instances can be a security risk. To protect publicly accessible RDS databases from brute force and dictionary attacks and add an additional layer of security, shift RDS instance ports to non-default ones.',
+    
+    link: 'https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_CommonTasks.Connect.html',
+    recommended_action: 'Change the default port number of the RDS instance to non-default port.',
+    apis: ['RDS:describeDBInstances'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        var defaultPorts = [
+            { 'engine': 'aurora-mysql', 'port': 3306 },
+            { 'engine': 'aurora-postgresql', 'port': 5432 },
+            { 'engine': 'custom-oracle-ee', 'port': 1521 },
+            { 'engine': 'mariadb', 'port': 3306 },
+            { 'engine': 'mysql', 'port': 3306 },
+            { 'engine': 'oracle-ee', 'port': 1521 },
```

Comment:
Why do we need redundancy? We can have just 1 value for oracle databases. Same goes for sqlserver